tests/integration/targets/rds_cluster_modify/tasks/remove_from_global_db.yaml

```diff
@@ -0,0 +1,244 @@
+---
+- name: Run tests for testing remove cluster from global db
+  block:
+
+    # Create global db -------------------------------------------------------------------------------
+
+    - name: Create rds global database
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ test_global_cluster_name }}"
+        engine: "{{ test_engine }}"
+        engine_version: "{{ test_engine_version }}"
+        region:  "{{ test_primary_cluster_region }}"
+        state: present
+      register: create_global_result
+
+    # Create primary cluster with an instance ---------------------------------------------------------------
+
+    - name: Create a primary cluster for global database
+      amazon.aws.rds_cluster:
+        global_cluster_identifier: "{{ test_global_cluster_name }}"
+        db_cluster_identifier: "{{ test_primary_cluster_name }}"
+        engine: "{{ test_engine }}"
+        engine_version: "{{ test_engine_version }}"
+        username: "{{ username }}"
+        password: "{{ password }}"
+      register: create_primary_result
+
+    - name: Create an instance connected to primary cluster
+      amazon.aws.rds_instance:
+        db_cluster_identifier: "{{ test_primary_cluster_name }}"
+        db_instance_identifier: "{{ test_primary_cluster_name }}-instance"
+        region: "{{ test_primary_cluster_region }}"
+        engine: "{{ test_engine }}"
+        db_instance_class: "{{ test_instance_class }}"
+
+    - name: Get primary cluster info
+      amazon.aws.rds_cluster:
+        global_cluster_identifier: "{{ test_global_cluster_name }}"
+        db_cluster_identifier: "{{ test_primary_cluster_name }}"
+      register: primary_cluster_info_result
```

Comment:
Can amazon.aws.rds_cluster_info module be used for this task? Seems like with the above task, amazon.aws.rds_cluster calls create_db_cluster and it fails with Parameter validation failed:\nMissing required parameter in input: \"Engine\"",